pyqir-jit/pyqir_jit/jit.py

```diff
@@ -4,14 +4,14 @@
 from typing import Any
 from .pyqir_jit import *
```

Comment:
Nit & optional: Import the items that are used in the code instead of the full library. Reason: If you have two or more of such statements, it becomes substantially harder to know where classes/functions live.